xbmc/video/VideoThumbLoader.cpp

```diff
@@ -557,6 +557,27 @@ void CVideoThumbLoader::DetectAndAddMissingItemData(CFileItem &item)
 {
   if (item.m_bIsFolder) return;
 
+  if (item.HasVideoInfoTag())
+  {
+    CStreamDetails& details = item.GetVideoInfoTag()->m_streamDetails;
+
+    // add audio language properties
+    for (int i = 1; i <= details.GetAudioStreamCount(); i++)
```

Comment:
Ah right index 0 returns the "best" stream. I've always hated that implementation.